force-app/main/releaseGating/classes/ReleaseGateVMapper.cls

```diff
@@ -156,6 +158,22 @@ public virtual with sharing class ReleaseGateVMapper {
         return featureVModelList;
     }
 
+    /**
+     * @description Verify if the result is successful, throw ReleaseGateCallableException if not
+     * @param  resultModel Callable action result
+     */
+    private void verifyCallableResult(CallableResult resultModel) {
+        if (resultModel.success) {
+            return;
+        } else {
```

Comment:
following what Matt told me about guard clauses, we could potentially change the 'else' to be another if. something like `if(resultModel.success == false){...}`

Reply:
In this case we don't even need the else condition.

Reply:
I copied this method from another class which Matt reviewed before.
The reason for this statement was to enforce code coverage for both true and false flows

Reply:
@iskander-sfdc I think Matt is referring to doing something like this:

```
    private void verifyCallableResult(CallableResult resultModel) {
        if (resultModel.success) {
            return;
        } 
            String errorMessage = resultModel.error != null
                ? String.format('{0} ({1})', new List<Object>{ resultModel.error.message, resultModel.error.code })
                : 'Unspecified error';
            UTIL_Debug.debug(System.LoggingLevel.ERROR, errorMessage);
            throw new ReleaseGateResultException(errorMessage);
        
    }
```

Reply:
thanks :-)